chalicelib/app_utils.py

```diff
@@ -75,15 +76,14 @@ def init_environments(env='all'):
     return environments
 
 
-def init_connection(environ, environments=None):
+def init_connection(environ):
     """
     Initialize the fourfront/s3 connection using the FSConnection object
     and the given environment.
     Returns an FSConnection object or raises an error.
     """
     error_res = {}
-    if environments is None:
-        environments = init_environments()
+    environments = init_environments(environ)
```

Comment:
Thanks for fixing, but the "description" in `error_res` still needs to be adjusted. Previously it would should all valid environments, but now it is misleading as written